### ISSUE TITLE
Support without 3rd party "prerender" service [feature request]

### ISSUE THREAD

**Issue by damianobarbati** [RisingStack/koa-prerender#15]
Would it be possible to support standalone prerendering (like `prerender-node` does with express) in current `koa-prerender`?

**Comment by tkodev** [comment]
Yes to this please. It would make this project usable if the 3rd party ever goes down.

**Comment by thoop** [comment]
this is already supported with the `prerender` option to pass the URL to the prerendering service.
